docs/src/pages/quasar-cli-vite/quasar-config-file.md

```diff
@@ -553,6 +553,7 @@ interface QuasarBuildConfiguration {
   /**
    * Sets [Vue Router mode](https://router.vuejs.org/guide/essentials/history-mode.html).
    * History mode requires configuration on your deployment web server too.
+   * For capacitor and electron, it's always 'hash' for [compatibility reasons](https://github.com/quasarframework/quasar/issues/17322#issuecomment-2191987962).
```

Comment:
Please also update the actual source files, which we usually copy into the docs.
Example:
https://github.com/quasarframework/quasar/blob/44d08caf1d15b8df4c576f0f4208f145e1436d59/app-vite/types/configuration/build.d.ts#L244-L250

Reply:
Also, capitalize Capacitor and Electron please
```suggestion
   * For Capacitor and Electron, it's always 'hash' for [compatibility reasons](https://github.com/quasarframework/quasar/issues/17322#issuecomment-2191987962).
```